lambdamud-client/src/components/MainScreen.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import axios from 'axios';
+import Pusher from 'pusher-js';
+
+var pusher = new Pusher('4467dc4bce09db57fe17', {
+  cluster: 'us2',
+  forceTLS: true
+});
+
+class MainScreen extends React.Component {
+  state = {
+    uuid: '',
+    name: '',
+    title: '',
+    description: '',
+    players: [],
+    error_msg: '',
+    userInput: '',
+  };
+
+  
+
+  componentDidMount() {
+    // gets token from local storage
+    const token = localStorage.getItem('token');
+
+    const options = {
+      headers: {
+        "Authorization": 'Token ' + token
+      }
+    }  
+
+  // GET axios call that initializes a player into a room
+  // it takes in the token to be able to do so
+  axios
+      .get('https://lambdamud-project-week.herokuapp.com/api/adv/init/', options)
+      .then(response => {
+        console.log(response);
+
+        var channel = pusher.subscribe('p-channel-' + response.data.uuid);
+  channel.bind('broadcast', function(message) {
+    alert(JSON.stringify(message));
+  });
+
+        this.setState({ uuid: response.data.uuid, name: response.data.name, title: response.data.title, description: response.data.description, players: response.data.players });
+      })
+      .catch(err => {
+        console.log(err.response);
+      });
+      
```

Comment:
Too much whitespace here, you need to remember to delete this after you delete code.